src/overlays/actors/ovl_En_Horse_Normal/z_en_horse_normal.c

```diff
@@ -1,15 +1,44 @@
+/*
+ * File: z_en_horse_normal.c
+ * Overlay: ovl_En_Horse_Normal
+ * Description: Non-rideable horses (Lon Lon Ranch and Stable)
+ */
+
 #include "z_en_horse_normal.h"
 
 #define FLAGS 0x00000000
 
 #define THIS ((EnHorseNormal*)thisx)
 
+typedef struct {
+    Vec3s pos;
+    u8 unk_06; // this may be a s16 if the always-0 following byte is actually not padding
+} EnHorseNormalUnkStruct1;
+
+typedef struct {
+    s32 len;
+    EnHorseNormalUnkStruct1* items;
+} EnHorseNormalUnkStruct2;
+
+typedef enum {
+    /* 0x00 */ HORSE_NULL, // Cycles through animations in sAnimations
+    /* 0x01 */ HORSE_WANDER,
+    /* 0x02 */ HORSE_STATIC,
+    /* 0x03 */ HORSE_STATIC_CLONEABLE,
+    /* 0x04 */ HORSE_FOLLOW_PATH
+} EnHorseNormalType;
```

Comment:
Based on the names you chose for action functions:
```suggestion
typedef enum {
    /* 0x00 */ HORSE_CYCLE_ANIMATIONS,
    /* 0x01 */ HORSE_WANDER,
    /* 0x02 */ HORSE_WAIT,
    /* 0x03 */ HORSE_WAIT_CLONE,
    /* 0x04 */ HORSE_FOLLOW_PATH
} EnHorseNormalAction;
```